Source/Core/Core/PowerPC/JitArm64/JitArm64_SystemRegisters.cpp

```diff
@@ -852,3 +852,44 @@ void JitArm64::mtfsfix(UGeckoInstruction inst)
   if (inst.CRFD == 7)
     UpdateRoundingMode();
 }
+
+void JitArm64::mtfsfx(UGeckoInstruction inst)
+{
+  INSTRUCTION_START
+  JITDISABLE(bJITSystemRegistersOff);
+  FALLBACK_IF(inst.Rc);
+
+  u32 mask = 0;
+  for (int i = 0; i < 8; i++)
+  {
+    if (inst.FM & (1 << i))
+      mask |= 0xF << (4 * i);
```

Comment:
```suggestion
      mask |= 0xFU << (4 * i);
```

Otherwise this is ORing a signed value